support-frontend/assets/components/otherAmount/otherAmount.tsx

```diff
@@ -9,7 +9,7 @@ const topSpacing = css`
 `;
 
 export type OtherAmountProps = {
-	selectedAmount: string;
+	selectedAmount: number | 'other';
```

Comment:
Where does 'other' come from? Would `number | undefined` maybe work better?

Reply:
'other' here is a valid value.
- `number` = the actual amount
- `other` = is for "Other / free text value"

We can definitely look at refactoring this out as a separate PR for clarity.